staging-installcheck.py

```diff
@@ -267,10 +267,21 @@ def staging(self, project, force=False):
                 result = False
 
         duplicates = duplicated_binaries_in_repo(self.api.apiurl, project, repository)
-        # remove white listed duplicates
+        # remove white listed duplicates and ignore patchinfos
         for arch in list(duplicates):
             for binary in self.ignore_duplicated:
                 duplicates[arch].pop(binary, None)
+
+            # In maintenance stagings there are patchinfo packages which collect binaries from
+            # other source packages. This leads to duplicate binaries being reported. So remove
+            # any patchinfos from the comparison.
+            for package in list(duplicates[arch]):
+                filtered = [s for s in duplicates[arch][package] if not s.startswith("patchinfo")]
```

Comment:
do they start with `patchinfo:` ? I personally would prefer to include the dot here at the end.